apiconcepts/integration/adding_custom_wizard_steps.md

```diff
@@ -0,0 +1,79 @@
+
+### ViewModel
+* The wizard infrastructure automatically assigns the `ViewModel` as DataContext to the view, so you can use `Binding` from the view to the ViewModel properties. If you don't need a ViewModel just assign `null` here.
+* For the `ViewModel` you can use any INotifyPropertyChanged implementation suits you best, such as providing your own or using an MVVM framework.
+
+### IsAvailable
+* This property is used by the wizard framework when navigating to other pages to determine if a page should be displayed. 
+* When the user clicks **Next** the wizard checks through the subsequent pages and displays the first one that is available. The **Next** button is disabled when there are no more pages for which `IsAvailable` returns `true`.
+* When the user clicks **Back** the wizard checks through the previous pages and displays the first one that is available. The **Back** button is disabled when there are no more pages for which `IsAvailable` returns `true`.
+
+### Data
+* You can define custom data for your pages using the `Data` property on the [StudioWizardPage](../../api/integration/Sdl.Desktop.IntegrationApi.Wizard.StudioWizardPage.yml) object.
+* The Data property is a `Dictionary<string,object>` that is common to all wizard pages. 
+* Once a dictionary key is set from a page, it becomes available to any pages that are displayed after it. This includes pages that are navigated to via the **Back** button. 
+* You can use the Page's `OnShow` method to query for the most recent key values that may have been updated/inserted by previously shown pages.
+* The `Data` property is initialized for all pages before the wizard is displayed.
+
+### Icon
+* Set this property to display an icon in the top-right area of the page.
+* Set this property to `null` if you don't want an icon displayed.
+
+### Id
+* This property is used by the wizard framework to uniquely identify the pages.
+
+  
+# Integrating pages as steps into <Var:ProductName> wizards
+Custom pages are injected into the wizard via the `firstPages` argument of the event object corresponding to the respective wizard. Keep in mind that the `firstPages` argument is available only for some wizard event objects.
+ 
+To inject the custom pages as steps into a <Var:ProductName> wizard:
+
+* Determine or define a place in your code to launch the wizard (with custom steps) from; for example, from an [Action](../../api/integration/Sdl.Desktop.IntegrationApi.AbstractAction.yml)'s `Execute` method.
+* Determine the appropriate event (e.g. [OpenProjectPackageEvent](../../api/integration/Sdl.TranslationStudioAutomation.IntegrationApi.Events.OpenProjectPackageEvent.yml)) to be raised, that corresponds to the wizard you want to open.
+* Set up a `List` of [StudioWizardPage](../../api/integration/Sdl.Desktop.IntegrationApi.Wizard.StudioWizardPage.yml) instances indicating the custom pages to be inserted
```

Comment:
Sounds a bit convoluted. Maybe phrase it something like... Set up a list of studioWizardPage instances, which represent the custom pages you want to insert in the wizard.

Reply:
Done.